src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -755,11 +756,22 @@ public Pod build(KubernetesSlave slave) {
         return new PodTemplateBuilder(this).withSlave(slave).build();
     }
 
+    /**
+     * Use getDescriptionForLogging(KubernetesSlave) instead
+     * @return
+     * @deprecated
+     */
+    @Deprecated
+    @Restricted(DoNotUse.class)
```

Comment:
```suggestion
```

This annotation has a specific purpose (technically `public` members which are not APIs). Do not misuse it as an extra emphasis for `@Deprecated`.